client/__tests__/e2e/e2eAnnotations.test.js

```diff
@@ -48,6 +84,87 @@ async function setup(config) {
   await waitByClass("autosave-complete");
 }
 
+describe("geneSET crud operations and interactions", () => {
+  test("genesets load from csv", async () => {
+    await goToPage(appUrlBase);
+
+    await assertGenesetExists(preExistingGenesetName);
+  });
+  test("brush on geneset mean", async () => {
+    await goToPage(appUrlBase);
+
+    await expandGeneset(meanExpressionBrushGenesetName);
+
+    const histBrushableAreaId = `histogram-${meanExpressionBrushGenesetName}-plot-brushable-area`;
+
+    const coords = await calcDragCoordinates(histBrushableAreaId, {
+      x1: 0.25,
+      y1: 0.5,
+      x2: 0.55,
+      y2: 0.5,
+    });
+
+    await drag(histBrushableAreaId, coords.start, coords.end);
+
+    const cellCount = await getCellSetCount(1);
+
+    expect(cellCount).toBe(meanExpressionBrushCellsSelected);
+  });
+  test("create a new geneset", async () => {
+    await goToPage(appUrlBase);
+
+    const genesetName = `test-geneset-foo-123`;
+    await assertGenesetDoesNotExist(genesetName);
+    await createGeneset(genesetName);
+    /* note: as of June 2021, the aria label is in the truncate component which clones the element */
+    await assertGenesetExists(genesetName);
+  });
+  test("edit geneset name", async () => {
```

Comment:
optional: you might consider linting this code, as it doesn't match our white space rules.